forc-pkg/Cargo.toml

```diff
@@ -41,9 +41,13 @@ tracing = "0.1"
 url = { version = "2.2", features = ["serde"] }
 vec1 = "1.8.0"
 walkdir = "2"
+fuel-asm = { version = "0.57.0", features = ["serde"] }
+sway-ir = { version = "0.63.5", path = "../sway-ir" }
+sysinfo = { version = "0.31.4", optional = true }
 
 [dev-dependencies]
 regex = "^1.10.2"
 
-[target.'cfg(not(target_os = "macos"))'.dependencies]
```

Comment:
I'm not convinced we can include `sysinfo` when on mac. We had it like this due to `sysinfo` causing compatibility issues with `fuel.nix` https://github.com/FuelLabs/fuel.nix/issues/64.

I'm happy for it to be included as long as we can confirm the build issue that was occurring in that issue has been resolved.

Reply:
I'm thinking we could probably narrow this further since it's only Apple Intel by `target_arch`?

Reply:
Thanks for pointing this out, it has been reverted to the previous implementation.